src/standalone/Selector/MultiSelectWithTags.tsx

```diff
@@ -90,8 +137,55 @@ const styles = createStyles((theme: Theme) => ({
 		margin: "5px",
 		lineHeight: "30px",
 	},
+	labelWithSwitch: {
+		marginTop: 15,
+	},
+	searchLabel: {
+		lineHeight: "30px",
+		float: "left",
+	},
+	switch: {
+		lineHeight: "30px",
+		float: "right",
+	},
 }));
 
+const AntSwitch = withStyles((theme: Theme) =>
+	createStyles({
```

Comment:
inline styles don't need createStyles. createStyles is only needed if you want to store the JSS in a variable